lib/ProductOpener/Config_off.pm

```diff
@@ -180,6 +180,7 @@ use ProductOpener::Config2;
 	stephane
 	tacinte
 	teolemon
+	himanshisrestha
```

Comment:
This is good for testing, but it should not be included in the PR.

Reply:
@himanshisrestha, when you test, use stephane or tacinte account to avoid having to change this file.

Reply:
@alexgarel could you please guide me on how to use it

Reply:
you just go to sign-in -- create account and use the username "stephane"